src/applications/claims-status/components/claim-letters/ClaimLetterListItem.jsx

```diff
@@ -62,7 +62,7 @@ const ClaimLetterListItem = ({ letter }) => {
         <span className="vads-u-display--block vads-u-font-size--h4 vads-u-margin-top--3 vads-u-margin-bottom--1">
           {getDescription(letter.docType)}
         </span>{' '}
-        <span className="vads-u-display--block vads-u-font-size--base vads-u-font-weight--normal vads-u-color--gray-warm-dark vads-u-line-height--4 vads-u-margin-bottom--0p5">
+        <span className="vads-u-display--block vads-u-font-size--base vads-u-font-weight--normal vads-u-color--gray-warm-dark vads-u-line-height--4 vads-u-margin-bottom--0p5 source-sans-font">
```

Comment:
We should be using the `vads-u-font-family--sans` utility class instead of custom css here. The reason it was not working before was because of an issue with the css library that was fixed last week. If you try using the utility class after pulling main and running `yarn install` again, you should see that it works